python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -404,6 +404,13 @@ class IsrTaskLSSTConfig(pipeBase.PipelineTaskConfig,
         default=True,
     )
 
+    # Flat correction.
+    doFlat = pexConfig.Field(
```

Comment:
@erykoff has mentioned that we are going to have multiple types of flats, not just one single flat. Does this placeholder refer to a particular flat?  is this the dome flat? Should we add comments to the code?

Reply:
I am adding a comment to say it would correspond to a background flat in this placeholder.